.travis.yml

```diff
@@ -101,7 +102,7 @@ before_script:
 script:
   - mkdir cmake-build
   - cd cmake-build
-  - cmake -DCMAKE_INSTALL_PREFIX:PATH="$HOME/OpenCoarrays" -DCMAKE_Fortran_FLAGS='-ftest-coverage -fprofile-arcs -O0' -DCMAKE_C_FLAGS='-fprofile-arcs -ftest-coverage -O0' ..
+  - cmake -DCMAKE_INSTALL_PREFIX:PATH="$HOME/OpenCoarrays" -DCMAKE_BUILD_TYPE="$BUILD_TYPE" ..
```

Comment:
Once I determine how, I would like to test also with the Release flags. Could also test OpenMPI *AND* MPICH